app/src/main/java/com/example/myapplication/ui/MainActivityScreen.kt

```diff
@@ -155,5 +162,41 @@ fun HorizontalScroll(
 }
 
 
+@Composable
+fun WidgetLayout(
```

Comment:
Should be private?

Reply:
Not yet

Reply:
This is view for global usage, not needed make it private